app/(public)/_components/RenderWebinar.tsx

```
@@ -0,0 +1,138 @@
+"use client";
+import React, { useEffect } from "react";
+
+import { type RenderWebinar } from "@/types";
+import { WebinarStatusEnum } from "@prisma/client";
+import WebinarUpcomingState from "@/components/WebinarUpcomingState";
+import { usePathname, useRouter } from "next/navigation";
+import { useAttendeeStore } from "@/hooks/useAttendeeStore";
+import { toast } from "sonner";
+import { XCircle, Calendar, AlertTriangle } from "lucide-react";
+
+const RenderWebinar = ({
```

Biome (2.1.2)

[error] Shouldn't redeclare 'RenderWebinar'. Consider to delete it or rename it.

'RenderWebinar' is defined here:


(lint/suspicious/noRedeclare)